pac.go

```diff
@@ -0,0 +1,71 @@
+package pac
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+var helperPath string = "./pac"
+
+// This library will extract a helper tool under application's same directory
+// to actually change proxy setup.
+// SetHelperPath specifies the file path to be generated.
+// It will be 'pac' under current work directory by default.
+func SetHelperPath(path string) {
+	helperPath = path
+}
+
+/* On tells OS to configure proxy through `pacUrl` */
+func On(pacUrl string) (err error) {
+	if err = ensureHelperTool(); err != nil {
+		err = fmt.Errorf("Unable to extract helper tool: %s", err)
+		return
+	}
+	cmd := exec.Command(helperPath, "on", pacUrl)
+	return run(cmd)
+}
+
+/* Off sets proxy mode back to direct/none */
+func Off() (err error) {
+	if err = ensureHelperTool(); err != nil {
+		err = fmt.Errorf("Unable to extract helper tool: %s", err)
+		return
+	}
+	cmd := exec.Command(helperPath, "off")
+	return run(cmd)
+}
+
+func run(cmd *exec.Cmd) error {
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return fmt.Errorf("Unable to execute pac tool: %s\n%s", err, out.String())
+	}
+	return nil
+}
+
+func ensureHelperTool() (err error) {
```

Comment:
That's a good reason :)  Do you mind switching to it?  It might actually help find some bugs in byteexec (in case there are any left hiding in there).